proxy_agent/src/telemetry/event_reader.rs

```diff
@@ -447,8 +457,10 @@ mod tests {
         assert!(THREAD_PRIORITY_VERIFY_DONE.load(Ordering::Relaxed));
     }
 
-    #[test]
-    fn test_event_reader_thread() {
+    // this test is to test the event reader thread, it reads events from the events folder and send to wire server
+    // it requires more threads to run server and client
```

Comment:
Do the tests hang or something with a single-thread runtime? They shouldn't, and if they do that's an indication that something's wrong with the code.